### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by danielsafari143**
In this pull request, I was able to : 
- Implement click event handlers using the math logic from calculate.js and operate.js 
- Make all calculator buttons work according to their functionality

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="3" />
+      <input type="button" onClick={handleSigns} className="signs orange" value="+" />
+      <input type="button" onClick={handleButtonClick} className="egals gray" value="0" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="." />
+      <input type="button" onClick={handleEqual} className="signs orange" value="=" />
+    </div>
+  );
+}
+
+export default Calculator;
```
Comment (julie-ify) [inline src/components/Calculator.jsx]: - [x] The Calculator component should pass props to at least one of the child components. You have a number of components like Egal, Input, and Sign, but none of them is being passed props from the Calculator. They are just redundant and not even being used in any other place in the app.

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <div className="calc">
+      <input value={btn} onChange={handleInput} className="inpute" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="A/C" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="+/-" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="%" />
+      <input type="button" onClick={handleSigns} className="signs orange" value="÷" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="7" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="8" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="9" />
+      <input type="button" onClick={handleSigns} className="signs orange" value="*" />
```
Comment (julie-ify) [inline src/components/Calculator.jsx]: - [x] I can't perform multiply operations. This should be "x", not "*"

  image

Comment (julie-ify) [inline src/components/Calculator.jsx]: - [ ] When performing calculations that involve dividing by 0 or modulo by 0 I should get the error messages in the operate.js file: "Can't divide by 0." and "Can't find modulo as can't divide by 0." But I get nothing. Kindly fix that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @danielsafari143,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @danielsafari143 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Chnages✔️
- Well Stractured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**